### PR TITLE
[12.0][ADD] crm_sale_secondary_salesperson: Crm Sale Secondary Salesperson

### PR DESCRIPTION
**Pull request by victoralmau**
Crm Sale Secondary Salesperson to unify `crm_secondary_salesperson` + `sale_secondary_salesperson`.

Related to https://github.com/OCA/sale-workflow/pull/1508

Please @joao-p-marques and @CarlosRoca13 can you review it?

@Tecnativa TT28664

### CODE REVIEW

File: crm_sale_secondary_salesperson/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module extends secondary salesperson from opportunity to sale order.
```
Comment (joao-p-marques) [inline crm_sale_secondary_salesperson/readme/DESCRIPTION.rst]:
```suggestion
This module extends the secondary salesperson from an opportunity to a sale order.
```

**Review by joao-p-marques — COMMENTED** [review]
LGTM :+1: 

Just a small comment regarding the README.

**Review by joao-p-marques — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]
LGTM

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**